src/scripts/core/models/token.ts

```diff
@@ -0,0 +1,64 @@
+import { Spine } from "pixi-spine";
+import { Assets, Container } from "pixi.js";
+import { eventEmitter } from "../../../event-emitter";
+//import { gsap } from 'gsap';
+
+
+export class Token extends Container {
+
+    private parentID: number;
+    private locationIndex;
+    private skin: Spine;
+    private skIndex: number;
+
+
+    constructor (parentID: number, locationIndex: number, size: number, skIndex: number, availWidth: number) {
+        super();
+
+        this.on('pointerdown', this.onClicked)
+        eventEmitter.on('tokenFirstClicked', this.onClicked);
+        eventEmitter.on('tokenSecondClicked', this.onClicked);
+
+
+        this.interactive = true;
+        this.parentID = parentID;
+        this.locationIndex = locationIndex;
+        this.skIndex = skIndex;
+        this.skin = new Spine(Assets.get('symbols').spineData);
+        this.skin.skeleton.setSkinByName('1');
```

Comment:
Guessing you don't need this line as the skin is set the proper value underneath?